languages/csharp/exercises/concept/nullability/Nullability.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+public static class Badge
+{
+    public static int ComputeWidthPx(string? text,
+                                     int? fontSizePx)
+    {
+        throw new NotImplementedException("Please implement the Badge.ComputeWidthPx method");
+    }
+
+    public static string ComputeNameText(string? firstName,
```

Comment:
Maybe rename this to `Label`:

```suggestion
    public static string Label(string? firstName,
```

You'd then have `Badge.Label` when calling it, which is quite nice.